mev_inspect/arbitrages.py

```diff
@@ -95,6 +95,7 @@ def _get_all_start_end_swaps(swaps: List[Swap]) -> List[Tuple[Swap, Swap]]:
                 == potential_end_swap.token_out_address
                 and potential_start_swap.from_address == potential_end_swap.to_address
                 and not potential_start_swap.from_address in pool_addrs
+                and potential_start_swap != potential_end_swap
```

Comment:
rather than comparing the objects directly which can sometimes be funky, I think it'd be slightly better to explicitly say what you mean which is "skip the swap I have as the start swap"

something like:
```
for index, potential_start_swap in enumerate(swaps):
    other_swaps = swaps[:index] + swaps[index+1:]
    for potential_end_swap in other_swaps:
        ...
```

Reply:
Thanks for the feedback, this was implemented